EIPS/eip-6384.md

```diff
@@ -0,0 +1,155 @@
+---
+eip: 6384
+title: Humanly readable offline signatures
+description: A procedure for making a proposed to be signed buffer of typed structured data humanly readable.
+author: Tal Be'ery (@talbeerysec), RoiV (@DeVaz1)
+discussions-to: https://ethereum-magicians.org/t/eip-6384-readable-eip-712-signatures/12752
+status: Draft
+type: Standards Track
+category: Interface
+created: 2023-01-08
+requires: 712
+---
+
+## Abstract
+
+This is a standard for making a proposed to be signed typed structured data humanly readable. It includes
+
+- Specification for including a function in the verifying contract that returns a humanly readable description of the message to be signed,
+- reference implementation,
+- Security analysis of the proposed solution
```

Comment:
The abstract should be a terse summary of the Specification section, perhaps with a tiny bit of background. Since every EIP follows the same format, you don't need to give an overview of the sections.

Perhaps something like:

```suggestion
This EIP introduces the `evalEIP712Buffer` function, which takes an [EIP-712](./eip-712.md) buffer and returns a human-readable text description.
```